checkClock/app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```

Comment:
If you don't occupy this helper, you can delete it.

Reply:
Ok, we will delete this helper